### ISSUE TITLE
Feeding Pet Bug

### ISSUE THREAD

**Issue by Tsukabi** [ossimc82/fabiano-swagger-of-doom#98]
The problem appears when I'm going to feed the pet which is following me. When I click on "feed pet", I can feed ONLY the pets in the Pet Yard, and not the one following me.
Thank you, bye.
-Tsukabi

**Comment by Sebafra-dev** [comment]
it was... close it ossimc :D

**Comment by Sebafra-dev** [comment]
https://github.com/ossimc82/fabiano-swagger-of-doom/issues/5

**Comment by ebf34e12952930cf** [comment]
Did you release some pets?
@sebastianfra12 #5 is a diffrent bug.

**Comment by Tsukabi** [comment]
No, i have only 2 pets, one in the graveyard, and the one witch is following me..

**Comment by ebf34e12952930cf** [comment]
@Tsukabi Try making a new account and see if you still get problems (if you do, provide some information)

**Comment by Tsukabi** [comment]
Thank you very much ! When the host will return online, I will try !
